app/containers/MessageBox/index.tsx

```diff
@@ -847,7 +841,12 @@ class MessageBox extends Component<IMessageBoxProps, IMessageBoxState> {
 			icon: 'discussions',
 			onPress: this.createDiscussion
 		});
-		showActionSheet({ options });
+
+		if (showEmojiKeyboard) {
+			this.closeEmoji();
+		}
+
+		setTimeout(() => showActionSheet({ options }), showEmojiKeyboard && isIOS ? 350 : null);
```

Comment:
Why do we need this timeout and the `isIOS`?
Can you use `requestAnimationFrame` or `InteractionManager.runAfterInteractions`?